src/routes/events.router.js

```diff
@@ -83,4 +83,39 @@ module.exports = (app) => {
      */
 
   app.get('/api/events', events.validate('auth'), events.findAll);
+
+  /**
+     * @swagger
+     * /api/events/verifyLocation:
+     *  get:
+     *      tags:
+     *          - Events
+     *      name: verify location
+     *      summary: Check if the user is in the event location
+     *      produces:
+     *          - application/json
+     *      consumes:
+     *          - application/x-www-form-urlencoded
+     *      parameters:
+     *          - name: eventId
```

Comment:
its says `eventId` but actually you use `eventsId` in code

Reply:
xD